data/gui/themes/default/dialogs/reachmap_options.cfg

```diff
@@ -0,0 +1,207 @@
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} lightblue}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} red}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} gold}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} brightgreen}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} brightorange}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} darkred}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} orange}
+			{_GUI_REACHMAP_COLOR_CELL {PURPOSE} green}
+		[/row]
+	[/grid]
+#enddef
+
+#define _GUI_REACHMAP_OPACITY_SLIDER_ROW PURPOSE LABEL TOOLTIP
+	[row]
+		[column]
+			horizontal_grow = true
+
+			[grid]
+				[row]
+					[column]
+						grow_factor = 0
+						border = "all"
+						border_size = 5
+						horizontal_alignment = "left"
+
+						[label]
+							label = {LABEL}
+						[/label]
+					[/column]
+
+					[column]
+						grow_factor = 1
+						border = "all"
+						border_size = 5
+						horizontal_grow = true
+
+						[slider]
+							id = "reachmap_opacity_{PURPOSE}"
+							definition = "default"
+							minimum_value = 0
+							maximum_value = 100
+							step_size = 1
+							tooltip = {TOOLTIP}
+						[/slider]
+					[/column]
+				[/row]
+			[/grid]
+		[/column]
+	[/row]
+#enddef
+
+[window]
+	id = "reachmap_options"
+	description = "Reach Map Options"
+
+	[resolution]
+		definition = "default"
+		click_dismiss = true
+		maximum_width = 600
+		maximum_height = 600
+
+		[tooltip]
+			id = "tooltip"
+		[/tooltip]
+
+		[helptip]
+			id = "tooltip"
+		[/helptip]
+
+		[grid]
+			[row]
+				[column]
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+					[label]
+						definition = "title"
+						label = _ "Reach Map Options"
+					[/label]
+				[/column]
+			[/row]
+			[row]
+				[column]
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+					[label]
+						definition = "default"
+						label = _ "Highlight Color"
+					[/label]
+				[/column]
+			[/row]
+	        [row]
+	        	[column]
+	        		horizontal_alignment = "left"
+	        		{_GUI_REACHMAP_COLOR_ROW standard_color}
+	        	[/column]
+	        [/row]
+	        {_GUI_REACHMAP_COLOR_SPACER_ROW}
+			[row]
+				[column]
+					border = "all"
+					border_size = 5
+					horizontal_alignment = "left"
+					[label]
+						definition = "default"
+						label = _ "Enemy Highlight Color"
+					[/label]
+				[/column]
+			[/row]
+	        [row]
+	        	[column]
+	        		horizontal_alignment = "left"
+	        		{_GUI_REACHMAP_COLOR_ROW enemy_color}
+	        	[/column]
+	        [/row]
+	        {_GUI_REACHMAP_COLOR_SPACER_ROW}
+
+	        {_GUI_REACHMAP_OPACITY_SLIDER_ROW border _"Border Opacity:" _"Change the opacity of the reach map border (percentage of the original reach image's opacity)"}
+
+	        {_GUI_REACHMAP_COLOR_SPACER_ROW}
+
+	        {_GUI_REACHMAP_OPACITY_SLIDER_ROW tint _"Tint Opacity:" _"Change the opacity of the reach map tint inside the border area (percentage of the original reach image's opacity)"}
+
+	        {_GUI_REACHMAP_COLOR_SPACER_ROW}
```

Comment:
Remove this local macro and replace it with `{GUI_FILLER_ROW HEIGHT=15}`

Reply:
ok, done